docs/sql-ref-syntax-qry-select-transform.md

```diff
@@ -0,0 +1,240 @@
+---
+layout: global
+title: TRANSFORM
+displayTitle: TRANSFORM
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+
+The `TRANSFORM` clause is used to specify a Hive-style transform query specification 
+to transform the inputs by running a user-specified command or script.
+
+### Syntax
+
+```sql
+SELECT TRANSFORM ( expression [ , ... ] )
+    [ ROW FORMAT row_format ]
+    [ RECORDWRITER record_writer_class ]
+    USING command_or_script [ AS ( [ col_name [ col_type ] ] [ , ... ] ) ]
+    [ ROW FORMAT row_format ]
+    [ RECORDREADER record_reader_class ]
+
+row_format:    
+    SERDE serde_class [ WITH SERDEPROPERTIES (k1=v1, k2=v2, ... ) ]
+    | DELIMITED [ FIELDS TERMINATED BY fields_terminated_char [ ESCAPED BY escaped_char ] ] 
+        [ COLLECTION ITEMS TERMINATED BY collection_items_terminated_char ] 
+        [ MAP KEYS TERMINATED BY map_key_terminated_char ]
+        [ LINES TERMINATED BY row_terminated_char ]
+        [ NULL DEFINED AS null_char ]
+```
+
+### Parameters
+
+* **expression**
+    
+    Specifies a combination of one or more values, operators and SQL functions that results in a value.
+
+* **row_format**    
+
+    Spark uses the `SERDE` clause to specify a custom SerDe for one table. Otherwise, use the `DELIMITED` clause to use the native SerDe and specify the delimiter, escape character, null character and so on.
```

Comment:
`Spark uses` -> `Uses`

Reply:
`Otherwise, use` -> `Otherwise, uses`

Reply:
To follow `**DELIMITED**`, `Otherwise, uses the `DELIMITED` clause to specify the native SerDe and state the delimiter, escape character, null character and so on.` ?
https://github.com/apache/spark/pull/31010/files#diff-6497e57374fd40ab3d88a249c83c858a52ab0440a1526e28c800ba706341396cR66

Reply:
done